### ISSUE TITLE
Show tooltip when focusing via keyboard

### ISSUE THREAD

**Issue by raffomania** [mishka-group/mishka_chelekom#408]
When I use the tab key on my keyboard to focus the "Notification" Button on the [tooltip example](https://mishka.tools/chelekom/docs/tooltip) page, the tooltip does not appear, and there is no focus indicator around the button. Expected behavior: I can see a focus ring around the button, and the tooltip is shown.

**Comment by MonaAghili** [comment]
Hi @raffomania thank you for reposting the issue
I'll definitely check it out and fix the problem

**Comment by raffomania** [comment]
Thanks!
